backend/src/infra/repositories/mysql/user-repository.ts

```diff
@@ -45,7 +46,12 @@ export class UserRepository
   }: LoadUserByEmail.Input): Promise<LoadUserByEmail.Output> {
     const userRepo = this.getRepository(User)
 
-    const user = await userRepo.findOne({ where: { email } })
+    const user = await userRepo
+      .createQueryBuilder('user')
+      .leftJoinAndSelect('user.role', 'role')
+      .leftJoinAndSelect('role.permissions', 'permission')
+      .where('user.email = :email', { email })
+      .getOne()
```

Comment:
Isso está quebrando por que não existe a tabela de permissions na `refactor/backend`

```suggestion
    const user = await userRepo.findOne({ where: { email } })
```